src/main/java/com/moodybluez/enterprise/dao/MoodDAOStub.java

```diff
@@ -15,12 +15,22 @@
 
     private Map<Integer, Mood> moods = new HashMap<>();
 
+/**
+ *
+ * @param mood
+ * @return mood
+ */
     @Override
     public Mood save(Mood mood) {
         moods.put(mood.getMoodId(), mood);
         return moods.get(mood.getMoodId());
     }
 
+/**
+ *
+ * @param moodId
+ * @return matchedMood
+ */
```

Comment:
needs explanation such as moodId this correlates to a specific mood in the database and
matchedMood returns the mood that correlates to the specific identifier passed in by moodId